spectral_connectivity/wrapper.py

```diff
@@ -0,0 +1,94 @@
+from spectral_connectivity.connectivity import Connectivity
+from spectral_connectivity.transforms import Multitaper
+import xarray as xr
+import numpy as np
+from logging import getLogger
+
+logger = getLogger(__name__)
+
+
+def connectivity_to_xarray(m, method='coherence_magnitude', signal_names=None, squeeze=False, **kwargs):
+    """
+    calculate connectivity using `method`. Returns an xarray
+    with dimensions of ['Time', 'Frequency', 'Source', 'Target']
+    or ['Time', 'Frequency'] if squeeze=True
+
+    signal_names : iterable of strings
+        Sames of time series used to name the 'Source' and 'Target' axes of xarray
+    squeeze : bool
+        Whether to only take the first and last source and target time series. Only makes sense for
+        one pair of signals and symmetrical measures
+
+    """
+    connectivity = Connectivity.from_multitaper(m, **kwargs)
+    connectivity_mat = getattr(connectivity, method)()
```

Comment:
kwargs should go here in the function call

Reply:
You can use this as a test case in the notebook. 

```

m = multitaper_connectivity(signal + noise,
                       sampling_frequency=sampling_frequency,
                       time_halfbandwidth_product=time_halfbandwidth_product,
                       time_window_duration=0.080,
                       time_window_step=0.080,
                       method='canonical_coherence',
                       connectivity_kwargs={"group_labels": group_labels}
                     )
```

Reply:
Hmm `canonical coherence` returns labels in addition to the matrix itself. There should be an if statement preceding `connectivity_mat = getattr(connectivity, method)(**kwargs)` unless you have a better suggestion. Are there other measures that have more than one output?

Reply:
Hmm, you're right. Canonical coherence should have the unique group labels as the targets and sources because it operates on groupings of signals. The other ones that have different outputs are: power, group_delay. Power is easy because it's just the signals. Group delay maybe just skip.

Reply:
The next commit skips canonical_coherence and group_delay